### PR TITLE
Russian language update

### PR DESCRIPTION
**Pull request by sli-pro**
- Update the Russian language.

### CODE REVIEW

File: mRemoteV1/Resources/Language/Language.ru.resx [hunk 1]
```diff
@@ -59,2208 +59,2240 @@
...
+	</data>
+	<data name="strCcICAOK" xml:space="preserve">
+		<value>Все компоненты ICA были найдены и зарегистрированы.
+Citrix ICA Client Control версии {0}</value>
+	</data>
+	<data name="strCcNotInstalledProperly" xml:space="preserve">
+		<value>установлен неправильно</value>
+	</data>
+	<data name="strCcPuttyFailed" xml:space="preserve">
+		<value>The SSH, Telnet, Rlogin and RAW protocols need PuTTY to work. PuTTY comes with every mRemoteNG package and is located in the installation path.
```
Comment (kmscode) [inline mRemoteV1/Resources/Language/Language.ru.resx]: This translation seems to have been removed?

File: mRemoteV1/Resources/Language/Language.ru.resx [hunk 2]
```diff
@@ -59,2208 +59,2240 @@
...
-  <data name="strMenuNewExternalTool" xml:space="preserve">
-    <value>Новый внешний инструмент</value>
-  </data>
-  <data name="strMenuNotifications" xml:space="preserve">
-    <value>Уведомления</value>
-  </data>
-  <data name="strMenuNotificationsCopyAll" xml:space="preserve">
-    <value>Скопировать все</value>
-  </data>
-  <data name="
```
Comment (kmscode) [inline mRemoteV1/Resources/Language/Language.ru.resx]: missing characters

Comment (kmscode) [inline mRemoteV1/Resources/Language/Language.ru.resx]: missing characters

Comment (kmscode) [inline mRemoteV1/Resources/Language/Language.ru.resx]: missing characters

**Comment by kmscode** [comment]
Thanks for the PR!

ConnectionTreeWindow.cs changes should be removed - I don't think we're enforcing method naming conventions strictly at this time. This will fix one of the conflicts as well.

Language.ru.resx_ needs to be removed.

Thanks again!

**Comment by sli-pro** [comment]
fixed

**Pull request closed without merging**